.github/workflows/reanimated-compatibility-check-nightly.yml

```
@@ -12,19 +12,19 @@ on:
 jobs:
   reanimated-compatibility-check-nightly:
     if: ${{github.repository == 'software-mansion/react-native-reanimated' && (github.event_name != 'pull_request' || contains(github.event.pull_request.labels.*.name, 'Check compatibility'))}}
-    runs-on: ${{matrix.platform == 'iOS' && 'macos-14' || 'ubuntu-latest'}}
+    runs-on: ${{matrix.platform == 'iOS' && 'macos-15' || 'ubuntu-latest'}}
     strategy:
       matrix:
         platform: ['iOS', 'Android']
         mode: ['Debug', 'Release']
         # TODO: Fetch compatibility in docs and here from common sources.
         # Update it when dropping support for RN versions.
-        react-native-version: ['0.78', '0.79', '0.80']
+        react-native-version: ['0.78', '0.79', '0.80', '0.81']
```

This will fail since version `0.81` can't be installed from NPM`. 

@skusnierz Is already working on this bit with reading the versions from a common package.json.

```suggestion
        react-native-version: ['0.78', '0.79', '0.80']
```

---

paper: https://github.com/software-mansion/react-native-reanimated/actions/runs/16781221822/job/47520111290
fabric: https://github.com/software-mansion/react-native-reanimated/actions/runs/16781221822/job/47520111349

Maybe they have already fixed the issue